### PR TITLE
GO-4118 research performance space waitload split subs

### PR DESCRIPTION
**Pull request by deff7**

### CODE REVIEW

File: core/indexer/reindex.go [hunk 1]
```diff
@@ -204,36 +202,9 @@ func (i *indexer) ReindexSpace(space clientspace.Space) (err error) {
 		}
 	}
 
-	i.addSyncDetails(space)
```
Comment (deff7) [inline core/indexer/reindex.go:207]: we're migrating from badger store, so sync details should be set automatically

File: core/block/object/objectcreator/installer.go [hunk 2]
```diff
@@ -141,6 +141,21 @@ func (s *service) listInstalledObjects(space clientspace.Space, sourceObjectIds
 				Condition:   model.BlockContentDataviewFilter_In,
 				Value:       pbtypes.StringList(sourceObjectIds),
 			},
+			{
+				Operator: model.BlockContentDataviewFilter_Or,
```
Comment (deff7) [inline core/block/object/objectcreator/installer.go:145]: utilize layout indexes

File: core/block/editor/archive.go [hunk 3]
```diff
@@ -77,7 +77,16 @@ func (p *Archive) updateObjects(_ smartblock.ApplyInfo) (err error) {
 	if err != nil {
 		return
 	}
+	go func() {
```
Comment (deff7) [inline core/block/editor/archive.go:80]: we can do it asynchronously

File: core/block/editor/dashboard.go [hunk 4]
```diff
@@ -75,6 +75,16 @@ func (p *Dashboard) updateObjects(info smartblock.ApplyInfo) (err error) {
 		return
 	}
 
+	go func() {
```
Comment (deff7) [inline core/block/editor/dashboard.go:78]: we can do it asynchronously

**Review by cheggaaa — APPROVED** [review]

**Pull request merged**